corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -746,20 +746,27 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", [
 
         performSubmit: function (initiatedBy) {
             var self = this;
-            self.validateAllFields().done(function () {
-                FormplayerFrontend.trigger(
-                    "menu:query",
-                    self.getAnswers(),
-                    self.options.sidebarEnabled,
-                    initiatedBy
-                );
-                if (self.smallScreenEnabled && self.options.sidebarEnabled) {
-                    $('#sidebar-region').collapse('hide');
-                }
-                sessionStorage.submitPerformed = true;
+            self.executeSearch(initiatedBy).done(function (response) {
+                self.updateModels(response);
+                self.displayErrors();
             });
         },
 
+        executeSearch: function (initiatedBy) {
+            var request = FormplayerFrontend.getChannel().request(
```

Comment:
I think this needs something like this:
```suggestion
            var self = this;
            var request = FormplayerFrontend.getChannel().request(
```
I'm getting this error on staging:
> Uncaught TypeError: self.getAnswers is not a function

Reply:
Thanks for catching, https://github.com/dimagi/commcare-hq/pull/34658/commits/bff1adcefee560f0c911bb36c09996af235e8f74